_about/FAQ.md

```diff
@@ -0,0 +1,40 @@
+# Frequently Asked Questions
+
+### Who are the maintainers?
+
+The HashiCorp Terraform Kubernetes provider team is :
+
+* Phil Sautter, Product Manager - [@redeux](https://github.com/redeux)
+* Alex Somesan, Engineer - [@alexsomesan](https://github.com/alexsomesan)
+* John Houston, Engineer - [@jrhouston](https://github.com/jrhouston)
+* Stef Forrester, Engineer - [@dak1n1](https://github.com/dak1n1)
+* Aareet Shermon, Engineering Manager - [@aareet](https://github.com/aareet)
+
+Our collaborators are:
+* Maximo Cuadros - [@mcuadros](https://github.com/mcuadros)
```

Comment:
@meyskens @mcuadros please let me know if you are comfortable being included in this list.

Reply:
I am no longer actively contributing here any longer as I changed employers

Reply:
From my side of course.

Reply:
No problem, thank you for all you've done so far. I'll update the list